samples/nrf5340/empty_net_core/README.rst

```diff
@@ -0,0 +1,33 @@
+.. _nrf5340_empty_net_core:
+
+nRF5340: Empty firmware for network core
+########################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+You can use this sample to generate empty network core firmware.
+
+Requirements
+************
+
+The sample supports the following development kit:
+
+.. table-from-sample-yaml::
+
+Building and running
+********************
+
+.. |sample path| replace:: :file:`samples/nrf5340/empty_net_core`
+
+.. include:: /includes/build_and_run.txt
+
+Testing
+=======
+
+The sample is used to prevent build failures when building MCUboot child image with nRF53 multi image update feature without network core child image in :ref:`ug_multi_image`.
+See the :ref:`thingy53_build_pgm_targets` section of Thingy:53 user guide for example use case.
```

Comment:
```suggestion
When building an MCUboot child image with the nRF53 multi-image update feature, the empty network core sample is built together with the application to prevent build failures related to missing Partition Manager definitions in the :ref:`ug_multi_image`.
The sample is used only by the applications that do not use the network core.
```